ParabolInc/parabol
# feat(kudos): send kudos by text in standups

Fixes #9230 

<img width="420" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/081daa96-0540-47aa-8795-f79e2e40d96b">

<img width="431" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/2e8df8cc-5aed-4e56-84d1-c876debdea75">

<img width="734" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/9ecd6262-28ab-431d-aa17-93ff792251f4">

<img width="708" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/069bfe60-779a-48c5-a66f-acf27841584b">



**How to test:**

- Add slack integration
- Start new standup meeting with few users
- Just mention a user, see mention notification was sent
- Edit response and add ❤️ emoji in the same paragraph where user was mentioned earlier. See that kudos notification was sent
- Edit response again, just add some text, see that no new notifications were sent
- Mention another user with ❤️ emoji, see that another user received kudos notification, but previous user didn't receive duplicate notification
- Check that slack notification for standup works and includes kudos
- Check that email notification for standup works and includes kudos
